pygmo/core.cpp

```diff
@@ -646,7 +651,8 @@ BOOST_PYTHON_MODULE(core)
     pygmo::expose_problem<tu_test_problem>("_tu_test_problem", "A thread unsafe test problem.");
     // Null problem.
     auto np = pygmo::expose_problem<null_problem>("null_problem", pygmo::null_problem_docstring().c_str());
-    np.def(bp::init<vector_double::size_type>((bp::arg("nobj"))));
+    np.def(bp::init<vector_double::size_type, vector_double::size_type, vector_double::size_type>(
```

Comment:
That was my worry. But I tried now and seems to work .... I put nobj to 1 by default. No idea which default constructor will be called